cg/server/endpoints/applications.py

```diff
@@ -23,6 +20,14 @@ def get_applications():
     return jsonify(applications=parsed_applications)
 
 
+@APPLICATIONS_BLUEPRINT.route("/applications/<order_type>")
+def get_application_order_types(order_type: str):
+    """Return application order types.."""
+    applications: list[Application] = db.get_applications_by_ordertype(order_type)
+    parsed_applications: list[dict] = [application.to_dict() for application in applications]
```

Comment:
```suggestion
    app_tags: list[str] = [application.tag for application in applications]
```
Do we need to return more data? In the current response, only the app tags are returned

Reply:
fixed